mbed_greentea/mbed_test_api.py

```diff
@@ -183,10 +189,15 @@ def run_command(cmd):
         gt_logger.gt_log_tab("calling mbedhtrun: %s"% " ".join(cmd))
     gt_logger.gt_log("mbed-host-test-runner: started")
 
-    htrun_output = ''
+    htrun_output = str()
     start_time = time()
 
+    # run_command will return None if process can't be opened (Issue #134)
     p = run_command(cmd)
+    if not p:
+        # int value > 0 notifies caller that starting of host test process failed
+        return 1729
```

Comment:
Yes, it is a special number: https://en.wikipedia.org/wiki/1729_%28number%29
It is just a value greater than zero, I will make it a const :)